.github/workflows/matchers/actionlint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+      {
+        "owner": "actionlint",
+        "pattern": [
+          {
+            "regexp": "^(?:\\x1b\\[\\d+m)?(.+?)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*: (?:\\x1b\\[\\d+m)*(.+?)(?:\\x1b\\[\\d+m)* \\[(.+?)\\]$",
+            "file": 1,
+            "line": 2,
+            "column": 3,
+            "message": 4,
+            "code": 5
+          }
+        ]
+      }
+    ]
+  }
```

Comment:
This file was copied directly from the [rhysd/actionlint](https://github.com/rhysd/actionlint) repo, as instructed in their documentation:

https://github.com/rhysd/actionlint/blob/main/docs/usage.md#problem-matchers

Reply:
Thanks Courtney! I've filed https://github.com/rhysd/actionlint/issues/520 as well to ask if the author would be open to improving this.